modules/local/ciriquant/ciriquant/main.nf

```diff
@@ -52,7 +52,7 @@ process CIRIQUANT {
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         bwa: \$(echo \$(bwa 2>&1) | sed 's/^.*Version: //; s/Contact:.*\$//')
-        ciriquant : \$(echo \$(CIRIquant --version 2>&1) | sed 's/CIRIquant //g' )
+        ciriquant: \$(echo \$(CIRIquant --version 2>&1) | sed 's/CIRIquant //g' )
         samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//')
         stringtie: \$(stringtie --version 2>&1)
         hisat2: $VERSION
```

Comment:
ok, so ciriquant has bwa, samtools, stringtie and hisat2 as well, that's interesting

Reply:
It actually requires all of these as [described here](https://ciri-cookbook.readthedocs.io/en/latest/CIRIquant_1_installation.html#prerequisites)